docs/contents/users/install/kubeadm-onsite.md

```diff
@@ -0,0 +1,164 @@
+cluster from your laptop:
+
+        cd /usr/bin
+        sudo rm kubelet kubeadm kubectl
+        sudo wget https://distro.eks.amazonaws.com/kubernetes-1-18/releases/1/artifacts/kubernetes/v1.18.9/bin/linux/amd64/kubelet; \
+        sudo wget https://distro.eks.amazonaws.com/kubernetes-1-18/releases/1/artifacts/kubernetes/v1.18.9/bin/linux/amd64/kubeadm; \
+        sudo wget https://distro.eks.amazonaws.com/kubernetes-1-18/releases/1/artifacts/kubernetes/v1.18.9/bin/linux/amd64/kubectl
+        chmod +x kubeadm kubectl kubelet
+
+1. To enable the *kubelet* service, type the following:
+
+        sudo systemctl enable kubelet
+
+1. Run the *kubeadm init* command, identifying the eks-distro image repository and Kubernetes version. Make any corrections requested for *kubeadm init* to complete successfully:
+
+        sudo kubeadm init --image-repository public.ecr.aws/eks-distro/kubernetes --kubernetes-version v1.18.9-eks-1-18-1
+        [init] Using Kubernetes version: v1.18.9-eks-1-18-1
+        [preflight] Running pre-flight checks
+        ...
+        [kubelet-finalize] Updating "/etc/kubernetes/kubelet.conf" to point to a rotatable kubelet client certificate and key
+        [addons] Applied essential addon: CoreDNS
+        [addons] Applied essential addon: kube-proxy
+        
+        Your Kubernetes control-plane has initialized successfully!
+
+    Your Kubernetes cluster should now be up and running. The kubeadm output should show the exact commands to use. If something goes wrong, correct the problem and run *kubeadm reset* to prepare you system to run *kubeadm init* again:
+
+1. Follow the instructions for configuring the client. To configure the client locally, type:
+
+        mkdir -p $HOME/.kube
+        sudo cp -i /etc/kubernetes/admin.conf $HOME/.kube/config
+        sudo chown $(id -u):$(id -g) $HOME/.kube/config
+
+1. Deploy a pod network to the cluster. See Installing Addons (https://kubernetes.io/docs/concepts/cluster-administration/addons) for information on available Kubernetes pod networks. For example, to deploy a Weaveworks network, type:
+
+        kubectl apply -f "https://cloud.weave.works/k8s/net?k8s-version=v1.18.9-eks-1-18-1"
+        serviceaccount/weave-net created
+        clusterrole.rbac.authorization.k8s.io/weave-net (http://clusterrole.rbac.authorization.k8s.io/weave-net) created
+        clusterrolebinding.rbac.authorization.k8s.io/weave-net (http://clusterrolebinding.rbac.authorization.k8s.io/weave-net) created
+        role.rbac.authorization.k8s.io/weave-net (http://role.rbac.authorization.k8s.io/weave-net) created
+        rolebinding.rbac.authorization.k8s.io/weave-net (http://rolebinding.rbac.authorization.k8s.io/weave-net) created
+        daemonset.apps/weave-net created
+
+    You can also consider [Calico](https://www.projectcalico.org/) or [Cillium](https://cilium.io/) networks. Calico is popular for on-premises use because Cisco switches talk to it easily.  
```

Comment:
maybe replace "Calico is popular for on-premises use because Cisco switches talk to it easily."

with

"Calico is popular because it can be used to propagate routes with BGP which is often used on-prem".

Reply:
Done.